### PR TITLE
fix(enumeration): remove items components if the items are empty

### PR DESCRIPTION
**Pull request by jare-talend**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
Items component are present if the items are items

**What is the new behavior?**

Items is only present if there are items

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://256.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://256.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://256.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://256.talend.surge.sh)

File: packages/components/src/Enumeration/Enumeration.component.js [hunk 1]
```diff
@@ -69,11 +69,12 @@ function Enumeration({ displayMode, headerDefault, headerInput,
 	return (
 		<div className={enumerationClasses()}>
 			{getHeader()}
-			<Items
-				items={items}
-				itemsProp={itemsProp}
-				currentEdit={currentEdit}
-			/>
+			{items.length > 0 &&
```
Comment (charlesnguyen) [inline packages/components/src/Enumeration/Enumeration.component.js]: It should be put in a function like getHeader() above.

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://256.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://256.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://256.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://256.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://256.talend.surge.sh)

**Pull request merged**